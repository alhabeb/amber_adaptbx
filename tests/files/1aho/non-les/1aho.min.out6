
          -------------------------------------------------------
          Amber 18 SANDER                              2018
          -------------------------------------------------------


|   Executable path: /Users/case/phenix/build/../conda_base/bin/sander
| Working directory: /Users/case/phenix/modules/amber_adaptbx/tests/files/1aho
|          Hostname: Unknown
  [-O]verwriting output

File Assignments:
|  MDIN: 1aho_amber_all.in                                                     
| MDOUT: 1aho.min.out                                                          
|INPCRD: 4amber_1aho.rst7                                                      
|  PARM: 4amber_1aho.prmtop                                                    
|RESTRT: 4amber_1aho.min.rst7                                                  
|  REFC: 4amber_1aho.rst7                                                      
| MDVEL: mdvel                                                                 
| MDFRC: mdfrc                                                                 
|  MDEN: mden                                                                  
| MDCRD: mdcrd                                                                 
|MDINFO: mdinfo                                                                
|  MTMD: mtmd                                                                  
|INPDIP: inpdip                                                                
|RSTDIP: rstdip                                                                
|INPTRA: inptraj                                                               


 Here is the input file:

Initial minimization                                                           
      &cntrl                                                                   
       ntwx   = 0, ntb    = 1, cut    = 9.0,     nsnb   = 10,                  
       imin   = 1, maxcyc = 50, ncyc   = 200, ntxo = 1,                        
       ntpr=10, ntr=1, restraint_wt=2.0, restraintmask='!@H=',                 
      /                                                                        
                                                                               

--------------------------------------------------------------------------------
   1.  RESOURCE   USE: 
--------------------------------------------------------------------------------

| Flags:                                                                        
 getting new box info from bottom of inpcrd
| peek_ewald_inpcrd: Box info found
|Largest sphere to fit in unit cell has radius =    15.050
| New format PARM file being parsed.
| Version =    1.000 Date = 05/20/20 Time = 11:04:10
 NATOM  =    5312 NTYPES =      16 NBONH =    3336 MBONA  =    2040
 NTHETH =    4068 MTHETA =    2760 NPHIH =    8584 MPHIA  =    8488
 NHPARM =       0 NPARM  =       0 NNB   =   22756 NRES   =     764
 NBONA  =    2040 NTHETA =    2760 NPHIA =    8488 NUMBND =      69
 NUMANG =     156 NPTRA  =     195 NATYP =      34 NPHB   =       1
 IFBOX  =       1 NMXRS  =      24 IFCAP =       0 NEXTRA =       0
 NCOPY  =       0


|     Memory Use     Allocated
|     Real              435536
|     Hollerith          16702
|     Integer           960277
|     Max Pairs        2356757
|     nblistReal         63744
|     nblist Int        276550
|       Total            18003 kbytes

| Note: 1-4 EEL scale factors are being read from the topology file.

| Note: 1-4 VDW scale factors are being read from the topology file.
| Duplicated    0 dihedrals
| Duplicated    0 dihedrals

     BOX TYPE: RECTILINEAR

--------------------------------------------------------------------------------
--------------------------------------------------------------------------------

default_name                                                                    

General flags:

Nature and format of input:
     ntx     =       1, irest   =       0, ntrx    =       1

Nature and format of output:
     ntxo    =       1, ntpr    =      10, ntrx    =       1, ntwr    =       1
     iwrap   =       0, ntwx    =       0, ntwv    =       0, ntwe    =       0
     ioutfm  =       0, ntwprt  =       0, idecomp =       0, rbornstat=      0

Potential function:
     ntf     =       1, ntb     =       1, igb     =       0, nsnb    =      10
     ipol    =       0, gbsa    =       0, iesp    =       0
     dielc   =   1.00000, cut     =   9.00000, intdiel =   1.00000

Frozen or restrained atoms:
     restraint_wt =   2.00000

Energy minimization:
     maxcyc  =      50, ncyc    =     200, ntmin   =       1

Ewald parameters:
     verbose =       0, ew_type =       0, nbflag  =       1, use_pme =       1
     vdwmeth =       1, eedmeth =       1, netfrc  =       0
     Box X =   45.900   Box Y =   40.700   Box Z =   30.100
     Alpha =   90.000   Beta  =   90.000   Gamma =   90.000
     NFFT1 =   48       NFFT2 =   40       NFFT3 =   30
     Cutoff=    9.000   Tol   =0.100E-04
     Ewald Coefficient =  0.30768
     Interpolation order =    4



   5.  REFERENCE ATOM COORDINATES

      
     Mask !@H=; matches  2492 atoms

--------------------------------------------------------------------------------
   3.  ATOMIC COORDINATES AND VELOCITIES
--------------------------------------------------------------------------------

                                                                                
 begin time read from input coords =     0.000 ps

 Number of triangulated 3-point waters found:      516

     Sum of charges from parm topology file =   3.99999985
     Assuming uniform neutralizing plasma

--------------------------------------------------------------------------------
   4.  RESULTS
--------------------------------------------------------------------------------

 ---------------------------------------------------
 APPROXIMATING switch and d/dx switch using CUBIC SPLINE INTERPOLATION
 using   5000.0 points per unit in tabled values
 TESTING RELATIVE ERROR over r ranging from 0.0 to cutoff
| CHECK switch(x): max rel err =   0.2738E-14   at   2.422500
| CHECK d/dx switch(x): max rel err =   0.8314E-11   at   2.736960
 ---------------------------------------------------
| Local SIZE OF NONBOND LIST =    1405744
| TOTAL SIZE OF NONBOND LIST =    1405744



 BOND    =      315.2742  ANGLE   =     3912.8077  DIHED      =     2701.5643
 VDWAALS =  3204672.8553  EEL     =   -19219.2109  HBOND      =        0.0000
 1-4 VDW =     1456.5446  1-4 EEL =    11401.8064  RESTRAINT  =        0.0000



 BOND    =      318.8429  ANGLE   =     3912.7762  DIHED      =     2701.5637
 VDWAALS =  2087209.0161  EEL     =   -19268.2138  HBOND      =        0.0000
 1-4 VDW =     1456.5422  1-4 EEL =    11401.8053  RESTRAINT  =        0.0214
 EAMBER  =  2087732.3327



 BOND    =      487.9155  ANGLE   =     3911.7108  DIHED      =     2701.5461
 VDWAALS =   385696.1048  EEL     =   -19532.2438  HBOND      =        0.0000
 1-4 VDW =     1456.4631  1-4 EEL =    11401.7680  RESTRAINT  =        1.1156
 EAMBER  =   386123.2644



 BOND    =     3456.7757  ANGLE   =     3255.3944  DIHED      =     2690.7562
 VDWAALS =     5568.9591  EEL     =   -19496.2044  HBOND      =        0.0000
 1-4 VDW =     1407.4130  1-4 EEL =    11375.7482  RESTRAINT  =       34.1226
 EAMBER  =     8258.8421



 BOND    =      536.3181  ANGLE   =      631.8330  DIHED      =     2678.7331
 VDWAALS =     -888.4223  EEL     =   -21495.9696  HBOND      =        0.0000
 1-4 VDW =     1072.7590  1-4 EEL =    11213.2470  RESTRAINT  =      127.5911
 EAMBER  =    -6251.5016



 BOND    =      303.9696  ANGLE   =      506.9573  DIHED      =     2685.8559
 VDWAALS =    -1321.5658  EEL     =   -22684.5564  HBOND      =        0.0000
 1-4 VDW =     1000.6063  1-4 EEL =    11168.8130  RESTRAINT  =      171.4031
 EAMBER  =    -8339.9201




                    FINAL RESULTS




 BOND    =      303.9696  ANGLE   =      506.9573  DIHED      =     2685.8559
 VDWAALS =    -1321.5658  EEL     =   -22684.5564  HBOND      =        0.0000
 1-4 VDW =     1000.6063  1-4 EEL =    11168.8130  RESTRAINT  =      171.4031
 EAMBER  =    -8339.9201

--------------------------------------------------------------------------------
   5.  TIMINGS
--------------------------------------------------------------------------------

|    Read coords time           0.00 ( 0.11% of Total)
|                Build the list             0.06 (90.32% of List )
|                Other                      0.01 ( 9.68% of List )
|             List time                  0.06 ( 4.46% of Nonbo)
|                   Short_ene time             1.13 (99.38% of Direc)
|                   Other                      0.01 ( 0.62% of Direc)
|                Direct Ewald time          1.14 (83.01% of Ewald)
|                Adjust Ewald time          0.03 ( 2.07% of Ewald)
|                Self Ewald time            0.00 ( 0.01% of Ewald)
|                   Fill Bspline coeffs        0.01 ( 5.91% of Recip)
|                   Fill charge grid           0.03 (13.80% of Recip)
|                   Scalar sum                 0.03 (14.97% of Recip)
|                   Grad sum                   0.05 (28.41% of Recip)
|                   FFT time                   0.07 (36.87% of Recip)
|                   Other                      0.00 ( 0.04% of Recip)
|                Recip Ewald time           0.19 (14.08% of Ewald)
|                Virial junk                0.00 ( 0.06% of Ewald)
|                Other                      0.01 ( 0.78% of Ewald)
|             Ewald time                 1.37 (95.54% of Nonbo)
|          Nonbond force              1.43 (93.81% of Force)
|          Bond/Angle/Dihedral        0.09 ( 6.08% of Force)
|          Other                      0.00 ( 0.12% of Force)
|       Force time                 1.53 (100.0% of Runmd)
|    Runmd Time                 1.53 (82.94% of Total)
| Total time                 1.84 (100.0% of ALL  )

| Number of list builds   :          2

| Highest rstack allocated:     273718
| Highest istack allocated:       5312
|           Job began  at 11:04:11.449  on 05/20/2020
|           Setup done at 11:04:11.505  on 05/20/2020
|           Run   done at 11:04:13.294  on 05/20/2020
