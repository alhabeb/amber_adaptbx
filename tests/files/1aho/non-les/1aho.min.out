
          -------------------------------------------------------
          Amber 16 SANDER                              2016
          -------------------------------------------------------

| Run on 03/01/2017 at 11:35:22

|   Executable path: sander
| Working directory: /Users/case/Dropbox/projects/xtal/test100/1aho_gap
|          Hostname: Unknown
  [-O]verwriting output

File Assignments:
|  MDIN: 1aho_amber_all.in                                                     
| MDOUT: 1aho.min.out                                                          
|INPCRD: 4amber_1aho.rst7                                                      
|  PARM: 4amber_1aho.prmtop                                                    
|RESTRT: 4amber_1aho.min.rst7                                                  
|  REFC: 4amber_1aho.rst7                                                      
| MDVEL: mdvel                                                                 
| MDFRC: mdfrc                                                                 
|  MDEN: mden                                                                  
| MDCRD: mdcrd                                                                 
|MDINFO: mdinfo                                                                
|  MTMD: mtmd                                                                  
|INPDIP: inpdip                                                                
|RSTDIP: rstdip                                                                
|INPTRA: inptraj                                                               


 Here is the input file:

Initial minimization                                                           
      &cntrl                                                                   
       ntwx   = 0, ntb    = 1, cut    = 9.0,     nsnb   = 10,                  
       imin   = 1, maxcyc = 50, ncyc   = 200, ntmin  = 1, ntxo = 1,            
       ntpr=10, ntr=1, restraint_wt=2.0, restraintmask='!@H=',                 
      /                                                                        
                                                                               

--------------------------------------------------------------------------------
   1.  RESOURCE   USE: 
--------------------------------------------------------------------------------

| Flags:                                                                        
 getting new box info from bottom of inpcrd
|  INFO: Old style inpcrd file read

| peek_ewald_inpcrd: Box info found
|Largest sphere to fit in unit cell has radius =    15.050
| New format PARM file being parsed.
| Version =    1.000 Date = 03/01/17 Time = 11:35:22
 NATOM  =    5312 NTYPES =      16 NBONH =    3336 MBONA  =    2040
 NTHETH =    4068 MTHETA =    2760 NPHIH =    8584 MPHIA  =    8488
 NHPARM =       0 NPARM  =       0 NNB   =   22756 NRES   =     764
 NBONA  =    2040 NTHETA =    2760 NPHIA =    8488 NUMBND =      69
 NUMANG =     156 NPTRA  =     195 NATYP =      34 NPHB   =       1
 IFBOX  =       1 NMXRS  =      24 IFCAP =       0 NEXTRA =       0
 NCOPY  =       0


|     Memory Use     Allocated
|     Real              523189
|     Hollerith          16702
|     Integer           960277
|     Max Pairs        2356757
|     nblistReal         63744
|     nblist Int        276550
|       Total            18688 kbytes

| Note: 1-4 EEL scale factors are being read from the topology file.

| Note: 1-4 VDW scale factors are being read from the topology file.
| Duplicated    0 dihedrals
| Duplicated    0 dihedrals

     BOX TYPE: RECTILINEAR

--------------------------------------------------------------------------------
   2.  CONTROL  DATA  FOR  THE  RUN
--------------------------------------------------------------------------------

default_name                                                                    

General flags:
     imin    =       1, nmropt  =       0

Nature and format of input:
     ntx     =       1, irest   =       0, ntrx    =       1

Nature and format of output:
     ntxo    =       1, ntpr    =      10, ntrx    =       1, ntwr    =       1
     iwrap   =       0, ntwx    =       0, ntwv    =       0, ntwe    =       0
     ioutfm  =       1, ntwprt  =       0, idecomp =       0, rbornstat=      0

Potential function:
     ntf     =       1, ntb     =       1, igb     =       0, nsnb    =      10
     ipol    =       0, gbsa    =       0, iesp    =       0
     dielc   =   1.00000, cut     =   9.00000, intdiel =   1.00000

Frozen or restrained atoms:
     ibelly  =       0, ntr     =       1
     restraint_wt =   2.00000

Energy minimization:
     maxcyc  =      50, ncyc    =     200, ntmin   =       1
     dx0     =   0.01000, drms    =   0.00010

Ewald parameters:
     verbose =       0, ew_type =       0, nbflag  =       1, use_pme =       1
     vdwmeth =       1, eedmeth =       1, netfrc  =       0
     Box X =   45.900   Box Y =   40.700   Box Z =   30.100
     Alpha =   90.000   Beta  =   90.000   Gamma =   90.000
     NFFT1 =   48       NFFT2 =   40       NFFT3 =   30
     Cutoff=    9.000   Tol   =0.100E-04
     Ewald Coefficient =  0.30768
     Interpolation order =    4

    LOADING THE CONSTRAINED ATOMS AS GROUPS


   5.  REFERENCE ATOM COORDINATES

      
     Mask !@H=; matches  2492 atoms

--------------------------------------------------------------------------------
   3.  ATOMIC COORDINATES AND VELOCITIES
--------------------------------------------------------------------------------

                                                                                
 begin time read from input coords =     0.000 ps

 Number of triangulated 3-point waters found:      516

     Sum of charges from parm topology file =   3.99999985
     Assuming uniform neutralizing plasma

--------------------------------------------------------------------------------
   4.  RESULTS
--------------------------------------------------------------------------------

 ---------------------------------------------------
 APPROXIMATING switch and d/dx switch using CUBIC SPLINE INTERPOLATION
 using   5000.0 points per unit in tabled values
 TESTING RELATIVE ERROR over r ranging from 0.0 to cutoff
| CHECK switch(x): max rel err =   0.2738E-14   at   2.422500
| CHECK d/dx switch(x): max rel err =   0.8314E-11   at   2.736960
 ---------------------------------------------------
| Local SIZE OF NONBOND LIST =    1405792
| TOTAL SIZE OF NONBOND LIST =    1405792


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
      1       3.3793E+06     1.2697E+05     4.2328E+06     O        1050

 BOND    =    58401.0114  ANGLE   =     4749.4462  DIHED      =     2786.7335
 VDWAALS =  3319681.5124  EEL     =   -19157.2652  HBOND      =        0.0000
 1-4 VDW =     1464.7075  1-4 EEL =    11361.7786  RESTRAINT  =        0.0000


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
     10       2.2585E+06     5.7312E+04     1.8518E+06     O        1050

 BOND    =    58402.2704  ANGLE   =     4749.4463  DIHED      =     2786.7055
 VDWAALS =  2198979.1346  EEL     =   -19205.4710  HBOND      =        0.0000
 1-4 VDW =     1464.7924  1-4 EEL =    11361.9553  RESTRAINT  =        0.0214
 EAMBER  =  2258538.8335


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
     20       5.0472E+05     6.8514E+03     1.4036E+05     O        5034

 BOND    =    58495.2970  ANGLE   =     4751.9715  DIHED      =     2786.1304
 VDWAALS =   445306.3140  EEL     =   -19452.4462  HBOND      =        0.0000
 1-4 VDW =     1466.6530  1-4 EEL =    11365.5618  RESTRAINT  =        1.1120
 EAMBER  =   504719.4813


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
     30       5.0394E+04     9.9438E+01     3.1726E+03     C        4872

 BOND    =    40050.1734  ANGLE   =     4466.9494  DIHED      =     2772.8661
 VDWAALS =     9592.0909  EEL     =   -19354.5188  HBOND      =        0.0000
 1-4 VDW =     1462.1796  1-4 EEL =    11369.2974  RESTRAINT  =       34.6807
 EAMBER  =    50359.0380


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
     40       1.1168E+03     2.4849E+01     8.8256E+02     C        4872

 BOND    =     5070.0650  ANGLE   =     2334.7665  DIHED      =     2785.9493
 VDWAALS =     -518.4835  EEL     =   -21050.4307  HBOND      =        0.0000
 1-4 VDW =     1087.6705  1-4 EEL =    11161.2032  RESTRAINT  =      246.0323
 EAMBER  =      870.7403


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
     50      -5.1357E+03     6.5655E+00     1.3758E+02     C         888

 BOND    =     1639.3030  ANGLE   =     1499.6414  DIHED      =     2845.0836
 VDWAALS =    -1142.5662  EEL     =   -22470.5325  HBOND      =        0.0000
 1-4 VDW =      999.7966  1-4 EEL =    11124.6876  RESTRAINT  =      368.8971
 EAMBER  =    -5504.5866


  Maximum number of minimization cycles reached.


                    FINAL RESULTS



   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
     50      -5.1357E+03     6.5655E+00     1.3758E+02     C         888

 BOND    =     1639.3030  ANGLE   =     1499.6414  DIHED      =     2845.0836
 VDWAALS =    -1142.5662  EEL     =   -22470.5325  HBOND      =        0.0000
 1-4 VDW =      999.7966  1-4 EEL =    11124.6876  RESTRAINT  =      368.8971
 EAMBER  =    -5504.5866

--------------------------------------------------------------------------------
   5.  TIMINGS
--------------------------------------------------------------------------------

|    Read coords time           0.00 ( 0.11% of Total)
|                Build the list             0.19 (95.87% of List )
|                Other                      0.01 ( 4.13% of List )
|             List time                  0.20 ( 9.24% of Nonbo)
|                   Short_ene time             1.65 (99.43% of Direc)
|                   Other                      0.01 ( 0.57% of Direc)
|                Direct Ewald time          1.66 (84.68% of Ewald)
|                Adjust Ewald time          0.05 ( 2.48% of Ewald)
|                Self Ewald time            0.00 ( 0.01% of Ewald)
|                   Fill Bspline coeffs        0.01 ( 4.98% of Recip)
|                   Fill charge grid           0.03 (12.95% of Recip)
|                   Scalar sum                 0.04 (17.19% of Recip)
|                   Grad sum                   0.06 (25.39% of Recip)
|                   FFT time                   0.09 (39.47% of Recip)
|                   Other                      0.00 ( 0.03% of Recip)
|                Recip Ewald time           0.23 (11.93% of Ewald)
|                Virial junk                0.00 ( 0.06% of Ewald)
|                Other                      0.02 ( 0.85% of Ewald)
|             Ewald time                 1.96 (90.76% of Nonbo)
|          Nonbond force              2.16 (94.14% of Force)
|          Bond/Angle/Dihedral        0.13 ( 5.81% of Force)
|          Other                      0.00 ( 0.06% of Force)
|       Force time                 2.30 (100.0% of Runmd)
|    Runmd Time                 2.30 (84.04% of Total)
|    Other                      0.43 (15.85% of Total)
| Total time                 2.73 (100.0% of ALL  )

| Number of list builds   :          5

| Highest rstack allocated:     273718
| Highest istack allocated:       5312
|           Job began  at 11:35:22.216  on 03/01/2017
|           Setup done at 11:35:22.297  on 03/01/2017
|           Run   done at 11:35:24.949  on 03/01/2017
|     wallclock() was called    1958 times
